### PR TITLE
Add a reset button to views

### PR DESCRIPTION
**Pull request by zackdavies**
Reset button added to both Events views. For https://github.com/localgovdrupal/localgov_events/issues/153

### CODE REVIEW

File: localgov_events.install [hunk 1]
```diff
@@ -73,3 +73,25 @@ function localgov_events_update_8002() {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Add Reset button to events views.
+ */
+function localgov_events_update_10001($is_syncing) {
```
Comment (ekes) [inline localgov_events.install]: Update hooks don't have a $is_syncing, they aren't called in syncing. They do have a sandbox argument - used for batching if I recall - but it's not required.

I again however feel queezy here pushing a button into live sites. How high are the chances of causing unexpected layout / presentation breakages for custom themes that have been built without it?

**Review by ekes — DISMISSED** [review]
For sure this will want the update hook correcting, but possibly removing.

**Comment by zackdavies** [comment]
OK, update job removed as requested.

**Review by andybroomfield — APPROVED** [review]
Happy to approve this.

**Pull request merged**